ui/app/components/clients/running-total.js

```diff
@@ -34,30 +34,24 @@ import { mean } from 'd3-array';
  * @param {array} barChartData - array of objects, object example: { month: '1/22', entity_clients: 11, non_entity_clients: 36, total: 47, namespaces: [] };
  * @param {array} chartLegend - array of objects with key names 'key' and 'label' so data can be stacked
  * @param {object} runningTotals - top level totals from /activity response { clients: 3517, entity_clients: 1593, non_entity_clients: 1924 }
+ * @param {string} timestamp -  string of ISO timestamp created in serializer to timestamp the response
  *   
  */
 export default class RunningTotal extends Component {
-  get getTotalClients() {
-    return (
-      this.args.chartLegend?.map((legend) => {
-        return {
-          label: legend.label,
-          total: this.args.runningTotals[legend.key],
-        };
-      }) || null
-    );
+  barChartData = this.args.barChartData;
```

Comment:
is there a benefit to making these variables here? could just get the mean from `this.args.barChartData`..

Reply:
Assigning an arg to a class property this way just doesn't quite seem right to me. For one thing if the arg changes that will not be reflected in `this.barChartData`. If you did want to assign the initial value to a property it would be more appropriate to do so in the constructor.

Reply:
Thanks! I tried that earlier and it didn't work - which is what prompted me reaching out in slack. Might have been some other syntax or typo issue as it works now! 🤯